Adding a CI workflow to re-build individual out-of-tree extensions

hannes opened the pull request:
So we can push bug fixes for extensions for already released DuckDB versions.

hannes committed:
attempt 1

hannes committed:
mc

hannes committed:
mc and moar platforms

hannes committed:
who invented yaml again?

hannes committed:
grrrr

hannes committed:
orr

hannes committed:
mc python ext

hannes committed:
no deps

hannes committed:
make cleanMerge branch 'master' of github.com:duckdb/duckdb into
extrebuild

hannes committed:
getting upload version right maybe

hannes committed:
need explicit checkout

hannes committed:
suure

hannes committed:
cant even

hannes committed:
re-adding workflows

hannes committed:
adding keys again

samansmink commented:
I feel there is quite some code duplication, but I don't really see how we would avoid that here, so LGTM!

hannes committed:
make cleanMerge branch 'master' of github.com:duckdb/duckdb into
extrebuild # Please enter a commit message to explain why this merge is
necessary, # especially if it merges an updated upstream into a topic
branch. # # Lines starting with '#' will be ignored, and an empty message
aborts # the commit.

hannes committed:
yay bash programming
